Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -879,6 +897,59 @@ def create_channel_command():
         demisto.results(f'The channel "{channel_name}" was created successfully')
 
 
+def create_meeting_command():
+    subject: str = demisto.args().get('subject', '')
+    start_date_time: str = demisto.args().get('start_date_time', '')
+    end_date_time: str = demisto.args().get('end_date_time', '')
+
+    meeting_data: dict = create_meeting(subject, start_date_time, end_date_time)
+    thread_id = ''
+    message_id = ''
+    chat_info = meeting_data.get('chatInfo', {})
+    if chat_info:
+        thread_id = chat_info.get('threadId', '')
+        message_id = chat_info.get('threadId', '')
+
+    participant_id, participant_display_name = get_participant_info(meeting_data.get('participants', {}))
+
+    outputs = {
+        'creationDateTime': meeting_data.get('creationDateTime', ""),
+        'threadId': thread_id,
+        'messageId': message_id,
+        'id': meeting_data.get('id', ""),
+        'joinWebUrl': meeting_data.get('joinWebUrl', ""),
+        'participantId': participant_id,
+        'participantDisplayName': participant_display_name
+    }
+    return CommandResults(
```

Comment:
why are you returning `CommandResult` ? `return_results` does not call this func